service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/ControlledAiNotebookInstanceResource.java

```diff
@@ -291,11 +291,11 @@ private static String mangleUsername(String username) {
     return mangledName;
   }
 
-  private static String extractUsername(@Nullable String validEmail) {
-    if (StringUtils.isEmpty(validEmail)) {
+  private static String extractInstanceName(@Nullable String validName) {
```

Comment:
remove since resourceName can't be nullable and this method is basically checking null now.